plugin/arcconf/arcconf.py

```diff
@@ -644,3 +644,49 @@ def volume_delete(self, volume, flags=0):
             raise LsmError(ErrorNumber.NOT_FOUND_VOLUME,
                            "Volume not found")
         return None
+
+    @_handle_errors
+    def volume_raid_info(self, volume, flags=Client.FLAG_RSVD):
+        """
+        :param volume: volume id - Volume to query
+        :param flags: optional. Reserved for future use.
+        :return: [raid_type, strip_size, disk_count, min_io_size, opt_io_size]
+
+        Depends on command:
+            arcconf getconfigjson <ctrlNo> array <arrayNo>
+        """
+
+        if not volume.plugin_data:
+            raise LsmError(
+                ErrorNumber.INVALID_ARGUMENT,
+                "Illegal input volume argument: missing plugin_data property")
+
+        volume_info = volume.plugin_data
+        ctrl_id = str(volume_info['ctrl_id'])
+        array_id = str(volume_info['array_id'])
+        stripe_size = int(volume_info['stripe_size']) * 1024  # convert to Kibibyte
```

Comment:
This might be `strip_size`.
And the `full_stripe_size` should be `stripe_size`.

You may refer to the document of `lsm.Client.volume_raid_info()` for detail.

Reply:
@cathay4t 
currently in arcconf property, there is no 'strip_size'.
but here 'strip_size = arcconf stripe_size' and 'stripe_size = arcconf full_stripe_size'